msm/src/mvlookup.rs

```diff
@@ -215,6 +220,46 @@ pub fn combine_lookups<F: Field, ID: LookupTableID>(
     lhs - rhs
 }
 
+/// Build the constraints for the lookup protocol.
+/// The constraints are the partial sum and the aggregation of the partial sums.
+#[allow(dead_code)]
+pub fn constraint_lookups<F: Field, ID: LookupTableID>(
+    lookups_map: &HashMap<ID, Vec<MVLookup<E<F>, ID>>>,
+) -> Vec<E<F>> {
+    let mut constraints: Vec<E<F>> = vec![];
+    let mut idx_partial_sum = 0;
+    lookups_map.iter().for_each(|(id, lookups)| {
+        let table_lookup = MVLookup {
+            table_id: *id,
+            numerator: -curr_cell(Column::LookupMultiplicity(id.to_u32())),
+            value: vec![curr_cell(Column::LookupFixedTable(id.to_u32()))],
```

Comment:
probably, this will only have an actual value once the joint combiner is chosen right?